Sources/ProjectSpec/Target.swift

```diff
@@ -279,8 +283,9 @@ extension Target: NamedJSONDictionaryConvertible {
         directlyEmbedCarthageDependencies = jsonDictionary.json(atKeyPath: "directlyEmbedCarthageDependencies")
         requiresObjCLinking = jsonDictionary.json(atKeyPath: "requiresObjCLinking")
 
-        prebuildScripts = jsonDictionary.json(atKeyPath: "prebuildScripts") ?? []
-        postbuildScripts = jsonDictionary.json(atKeyPath: "postbuildScripts") ?? []
+        preBuildScripts = jsonDictionary.json(atKeyPath: "preBuildScripts") ?? jsonDictionary.json(atKeyPath: "prebuildScripts") ?? []
```

Comment:
Makes me think the `lint` command in https://github.com/yonaskolb/XcodeGen/issues/214 could also be used to highlight deprecations